dcos_installer/cli.py

```diff
@@ -202,6 +202,12 @@ def dispatch(args):
         sys.stdout.buffer.write(byte_str + b'\n')
         sys.exit(0)
 
+    if args.action == 'generate-node-upgrade-script':
+        messages = backend.generate_node_upgrade_script(args.installed_version)
+        if messages:
+            sys.exit(1)
+        sys.exit(0)
```

Comment:
Shouldn't this just be returning the exit code returned by `generate_node_upgrade_script()`?